02-Logging/01-Displaying-Status-Information.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>This is less a question about the tutorial and more about the design: is there a reason that this thing is named <code>flag</code>  instead of <code>category</code>  or <code>type</code>?  I would avoid <code>type</code>  as it is a built-in keyword, but <code>category</code>  seems much more apt than <code>flag</code>, which to me implies some sort of go/no-go.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>That is a question for @b-butler.</p>

Reply:
<p>I named it flag because it flagged the type of the loggable. Category works as well, it just wasn't the term I thought of at the time. The enum used internally is essentially a bunch of bit flags.</p>

Reply:
I think it's more semantically clear to use something other than flag; the underlying implementation detail that it's bit flags is less important than the fact that it's a description of the type of data. When I see flag I have different expectations of what it would mean.